cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -956,6 +956,8 @@
       public static final String EXTENSIONS_DIR = "security.store.extensions.dir";
       /**Secure store extension property prefix*/
       public static final String PROPERTY_PREFIX = "security.store.system.properties.";
+      /** Secure store service*/
+      public static final String SECURE_STORE_SERVICE = "appfabric";
```

Comment:
Shouldn't use "appfabric" as the name. Instead, define an appropriate name for it and then add the name binding in `AppFabricServiceRuntimeModule`.